dtbase/backend/api/location/routes.py

```diff
@@ -148,3 +148,63 @@ def list_locations(schema_name):
         session=db.session,
     )
     return jsonify(result), 200
+
+
+@blueprint.route("/list_location_schemas", methods=["GET"])
+# @login_required
+def list_location_schemas():
+    """
+    List location schemas in the database.
+    """
+
+    result = locations.list_location_schemas(session=db.session)
+    return jsonify(result), 200
+
+@blueprint.route("/list_location_identifiers", methods=["GET"])
+# @login_required
+def list_location_identifiers():
+    """
+    List location identifiers in the database.
+    """
+
+    result = locations.list_location_identifiers(session=db.session)
+    return jsonify(result), 200
+
+
+@blueprint.route("/delete_location_schema/<schema_name>", methods=["DELETE"])
+# @login_required
+def delete_location_schema(schema_name):
+    """
+    Delete a location schema from the database.
+    Endpoint URL: /delete_location_schema/<schema_name>
+    """
+
+    locations.delete_location_schema(schema_name=schema_name, session=db.session)
+    db.session.commit()
+
+    # Check if the schema still exists in the database
```

Comment:
Do we need this test to be run every time the API endpoint gets called? I might have this just as a test in the test suite.

Reply:
If the issue is that `locations.delete_location_schema` fails silently if we try to e.g. delete a non-existent schema, then maybe that could be dealt with in the `locations` module without an extra database query somehow?

Reply:
Yes, that makes sense. I removed it and now return a boolean from the locations module to check if rows have been deleted.